### ISSUE TITLE
Running Terraform gives error: 'Error: API (405): HTTP method POST is required'

### ISSUE THREAD

**Issue by esrahofstede** [rewe-digital/terraform-provider-sonarcloud#20]
While trying to run this SonarCloud Terraform Provider version 0.1.0, the following errors occur:
```
2021-07-14 18:02:36,629 - ERROR - Error: API (405): HTTP method POST is required
2021-07-14 18:02:36,630 - ERROR -   on groups.tf line 4, in resource "sonarcloud_user_group" "user_group_REDACTED_GROUP":
2021-07-14 18:02:36,630 - ERROR -    4: resource "sonarcloud_user_group" "user_group_REDACTED_GROUP" {
2021-07-14 18:02:36,635 - ERROR - Error: API (405): HTTP method POST is required
2021-07-14 18:02:36,635 - ERROR -   on groups.tf line 46, in resource "sonarcloud_user_group_member" "group_member_REDACTED_GROUP_REDACTED_USER":
2021-07-14 18:02:36,635 - ERROR -   46: resource "sonarcloud_user_group_member" "group_member_REDACTED_GROUP_REDACTED_USER" {
2021-07-14 18:02:36,636 - ERROR - Terraform encountered an error.
```
The result is no terraform resources are created.
This led to testing locally running `make testacc`. Unfortunately the same results were seen running the Acceptance Tests:
```
--- FAIL: TestAccDataSourceUserGroups (0.27s)
=== RUN   TestAccUserGroupMember
    resource_user_group_member_test.go:16: SONARCLOUD_TEST_GROUP_NAME must be set for acceptance tests
--- FAIL: TestAccUserGroupMember (0.00s)
=== RUN   TestAccUserGroupPermissions
    resource_user_group_permissions_test.go:21: Step 1/6 error: Error running apply: exit status 1
        Error: API (405): HTTP method POST is required
          with sonarcloud_user_group_permissions.test_permission,
          on terraform_plugin_test.tf line 2, in resource "sonarcloud_user_group_permissions" "test_permission":
           2: resource "sonarcloud_user_group_permissions" "test_permission" {
--- FAIL: TestAccUserGroupPermissions (1.38s)
=== RUN   TestAccUserGroup
    resource_user_group_test.go:14: Step 1/3 error: Error running apply: exit status 1
        Error: API (405): HTTP method POST is required
          with sonarcloud_user_group.test_group,
          on terraform_plugin_test.tf line 2, in resource "sonarcloud_user_group" "test_group":
           2: resource "sonarcloud_user_group" "test_group" {
        
--- FAIL: TestAccUserGroup (0.39s)
=== RUN   TestAccUserToken
    resource_user_token_test.go:21: Step 1/1 error: Error running apply: exit status 1
        Error: API (405): HTTP method POST is required
          with sonarcloud_user_token.test_token,
          on terraform_plugin_test.tf line 2, in resource "sonarcloud_user_token" "test_token":
           2: resource "sonarcloud_user_token" "test_token" {
--- FAIL: TestAccUserToken (0.40s)
FAIL
FAIL    terraform-provider-sonarcloud/sonarcloud        3.081s
FAIL
Makefile:26: recipe for target 'testacc' failed
make: *** [testacc] Error 1
```

Pull request #19 was created for making the `make testacc` run more smoothly, although it might no be necessary anymore because the terraform dependencies are already updated on the main branch

**Comment by reinoudk** [comment]
Hi @esrahofstede,

Thanks for creating an issue. I'll try and reproduce this on our side. It might be possible that that underlying API has changed.

**Comment by reinoudk** [comment]
This should be fixed in `v0.1.1`.

**Comment by esrahofstede** [comment]
I have updated our project and these changes fix this issue
